docs/installation.rst

```diff
@@ -57,11 +57,48 @@ For example: ::
         poldracklab/fmriprep:latest \
         /data /out/out \
         participant \
-        --ignore fieldmaps
+        --ignore fieldmaps --no-freesurfer
 
 See `External Dependencies`_ for more information (e.g., specific versions) on
 what is included in the latest Docker images.
 
+If the flag ``--no-freesurfer`` is not set, then FreeSurfer will require a proper
+license file.
+It is possible to run the docker container pointing the image to a local path
+where a valid license file is stored.
+For example, if the license is stored in the ``$HOME/.licenses/freesurfer/license.txt`` 
+file on the host system: ::
+
+    $ docker run -ti --rm \
+        -v $HOME/fullds005:/data:ro \
+        -v $HOME/dockerout:/out \
+        -v $HOME/.licenses/freesurfer/license.txt:/etc/licenses/freesurfer/license.txt \
+        poldracklab/fmriprep:latest \
+        /data /out/out \
+        participant \
+        --ignore fieldmaps
+
+It is also possible to run ``fmriprep-docker`` with FreeSurfer processing: ::
+
+    $ fmriprep-docker --fs-license-file $HOME/.licenses/freesurfer/license.txt \
+        /path/to/data/dir /path/to/output/dir participant
+    RUNNING: docker run --rm -it -v /path/to/data/dir:/data:ro \
+        -v /path/to_output/dir:/out poldracklab/fmriprep:1.0.0 \
```

Comment:
This command doesn't have a `-v /home/user/.licenses/freesurfer/license.txt:/etc/licenses/freesurfer/license.txt` line (or similar.